### PR TITLE
Don't write to defaults dir but to local

### PR DESCRIPTION
**Pull request by shaharke**
@hkariti how do I avoid text dup?

### CODE REVIEW

**Comment by hkariti** [comment]
what text dup?

**Comment by shaharke** [comment]
look at the ####BigPanda### that appears in two different functions. Inside the tee command.

File: bin/bigpanda-splunk-defaults [hunk 1]
```diff
@@ -54,41 +63,42 @@ while true; do
...
-        exit_after_chmod 1
+        if grep -i --silent "action.script\s*=\s*1" $CONFIGURATION_FILE; then
+            echo "WARN:  action.script is already enabled."
+        else
+            echo "INFO:  Enabling action.script by default for all alerts"
+            sudo -u splunk sed -i.bak '/^action.script *= *0$/ s/0/1/' $CONFIGURATION_FILE
+        fi
+    fi
+
+    # Make sure action.script.filename was not already configured
```
Comment (shaharke) [inline bin/bigpanda-splunk-defaults:89]: @hkariti can you take a quick look?

File: bin/bigpanda-splunk-defaults [hunk 2]
```diff
@@ -54,41 +63,42 @@ while true; do
...
+    if grep -i --silent "action.script.filename" $CONFIGURATION_FILE; then
```
Comment (hkariti) [inline bin/bigpanda-splunk-defaults:90]: Why is this not 'else' on the prev if?

File: bin/bigpanda-splunk-defaults [hunk 3]
```diff
@@ -54,41 +63,42 @@ while true; do
...
-# Make sure action.script.filename was not already configured
-if grep -i --silent "action.script.filename" $CONFIGURATION_FILE; then
-    # Make sure we're not grepping our own configuration
-    if grep -i --silent "action.script.filename\s*=.*bigpanda.*" $CONFIGURATION_FILE; then
-        already_configured
+    # Enable the script action by default for all alerts (existing and new)
+    if ! grep -i --silent "action.script\s*=" $CONFIGURATION_FILE; then
+        echo "INFO:  BLA"
+        echo "INFO:  Enabling action.script by default for all alerts"
+        echo "action.script = 1" >> $CONFIGURATION_FILE
```
Comment (hkariti) [inline bin/bigpanda-splunk-defaults:79]: permissions?

File: bin/bigpanda-splunk-defaults [hunk 4]
```diff
@@ -54,41 +63,42 @@ while true; do
...
-fi
 
-# Make sure action.script.filename was not already configured
-if grep -i --silent "action.script.filename" $CONFIGURATION_FILE; then
-    # Make sure we're not grepping our own configuration
-    if grep -i --silent "action.script.filename\s*=.*bigpanda.*" $CONFIGURATION_FILE; then
-        already_configured
+    # Enable the script action by default for all alerts (existing and new)
+    if ! grep -i --silent "action.script\s*=" $CONFIGURATION_FILE; then
+        echo "INFO:  BLA"
```
Comment (hkariti) [inline bin/bigpanda-splunk-defaults:77]: maybe want to remove this :P

Comment (shaharke) [inline bin/bigpanda-splunk-defaults:77]: oops

File: bin/bigpanda-splunk-defaults (hunk 3 above)
Comment (shaharke) [inline bin/bigpanda-splunk-defaults:79]: if the script runs with sudo it should be ok no?

File: bin/bigpanda-splunk-defaults (hunk 2 above)
Comment (shaharke) [inline bin/bigpanda-splunk-defaults:90]: it's a different regex

File: bin/bigpanda-splunk-defaults (hunk 3 above)
Comment (hkariti) [inline bin/bigpanda-splunk-defaults:79]: yeah it should. do you check you are though?
Also, you use sudo | tee in the rest of the script

File: bin/bigpanda-splunk-defaults [hunk 5]
```diff
@@ -54,41 +63,42 @@ while true; do
...
+        echo "INFO:  Enabling action.script by default for all alerts"
+        echo "action.script = 1" >> $CONFIGURATION_FILE
     else
-        echo "ERROR:  action.script.filename is already configured. Contact support@bigpanda.io for details on how to configure an existing action script."
-        exit_after_chmod 1
+        if grep -i --silent "action.script\s*=\s*1" $CONFIGURATION_FILE; then
+            echo "WARN:  action.script is already enabled."
```
Comment (hkariti) [inline bin/bigpanda-splunk-defaults:82]: Why warn?

**Pull request merged**